physionet-django/project/utility.py

```diff
@@ -677,3 +679,42 @@ def convert_url(self, attr_name, url):
             if newurl != url:
                 return newurl
         return url
+
+
+class DataSourceCreator:
+    def __init__(self, **kwargs):
+        self.data_location = kwargs.get('data_location', None)
+        self.files_available = kwargs.get('files_available', None)
+        self.email = kwargs.get('email', None)
+        self.uri = kwargs.get('uri', None)
+        self.access_mechanism = kwargs.get('access_mechanism', None)
+
+    def create(self, project):
+        DataSource.objects.create(
+            project=project,
+            files_available=self.files_available,
+            data_location=self.data_location,
+            access_mechanism=self.access_mechanism,
+            email=self.email,
+            uri=self.uri,
+        )
+
+    @staticmethod
```

Comment:
I know `self` is not needed here, but this is called on an instance everywhere (see your uses of this method - you first instantiate the class). Maybe `@staticmethod` is not necessary here.